gluten-core/src/main/scala/io/glutenproject/extension/columnar/RewriteTypedImperativeAggregate.scala

```diff
@@ -40,6 +41,38 @@ object RewriteTypedImperativeAggregate extends Rule[SparkPlan] with PullOutProje
     }
   }
 
+  def shouldRewriteForPercentileLikeExpr(ae: AggregateExpression): Boolean = {
+    ae.aggregateFunction match {
+      case _: ApproximatePercentile =>
+        ae.mode match {
+          case Partial | PartialMerge => true
```

Comment:
It only support `Partial` and `Final` in `HashAggregateExecTransformer.scala`. Shall we add `PartialMerge` here?

Reply:
yeah, it should be removed